04week/loops.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+var cars = ['Ford','Dodge','Chevy','Mazda'];
+var moreCars = ['Toyota','Nissian','Lexus','Honda'];
+let totalCars = cars.concat(moreCars);
+var stringOfCars = totalCars.join();
+totalCars = stringOfCars.split(",");
+var carsInReverse = totalCars.reverse();
+carsInReverse = carsInReverse.sort();
+// alert(carsInReverse.indexOf('Chevy'));
+var removedCars = carsInReverse.slice(2,4);
+carsInReverse.splice(1,2);
+carsInReverse.push("Dodge", "Ford");
+carsInReverse.pop();
+carsInReverse.shift();
+carsInReverse.unshift("Buick");
+var numbers = [23,45,0,2];
+numbers.forEach(function (value,index, numbers){
+  numbers[index] = value +2;
+});
+console.log('carsInReverse: '+carsInReverse);
+for (var i=0; i<carsInReverse.length; i++) {
+  let car = carsInReverse[i];
+  console.log(car);
+}
+let persons = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+}
+var text = "";
+var x;
+}
+for (x in persons) {
```

Comment:
There was a task to do a check on the key, and print out the value if the key is a birthdate